Makefile

```diff
@@ -161,6 +161,17 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+YAMLS_DIRS ?= $(ROOT_DIR)/deploy/examples
+YAMLLINT_CONF ?= $(ROOT_DIR)/.yamllint
```

Comment:
I don't foresee any need for users to edit these values when running locally, or for us to modify them in CI. These might be best set inline to avoid the complexity of maintaining configurability. If someone needs the configurability, they will probably tell us.

Reply:
I removed the variables even though I generally like to add them, not specifically but for better readability (IMHO).